content/docs/packages/contribution_guidelines.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
This is great, but I don't think this belongs here.

I think all this info should be on the contributors guide on GitHub that we already have.

There's two big types of contributions:
- Changes to the core code base that this, an our existing contributor guidelines are helpful towards
- New Dataform packages

This section of the docs site should really be about the latter. Reality is we don't know what the right guidelines are yet for packages, but Dan and I made the first few last week so we can probably fill out this page soon! Feel free to leave it blank for now and copy what you've done here into the  existing contributor guide

Reply:
Aah I see, that makes sense. I will move it to there.